sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// Represents a Selection mark recognized from the input document.
+    /// </summary>
+    public class FormSelectionMark : FormElement
+    {
+        internal FormSelectionMark(SelectionMark selectionMark, int pageNumber)
+            : base(new FieldBoundingBox(selectionMark.BoundingBox), pageNumber, selectionMark.State.ToString())
```

Comment:
Yes. we are all doing it.
The service only returns information about the checkbox, not the text attached to it. although because this is only for labeled models, the user can access that text too.
If we leave the property empty then we wouldn't be able to access the value of the selection mark when [looking](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample2_RecognizeCustomForms.md#recognize-custom-forms-from-a-uri) at the `RecognizedForm.Fields.ValueData` i.e. `Console.WriteLine($"    Value: '{field.ValueData.Text}");`